src/main/java/com/hubspot/jinjava/lib/fn/Functions.java

```diff
@@ -1,6 +1,8 @@
 package com.hubspot.jinjava.lib.fn;
 
+import static com.hubspot.jinjava.interpret.JinjavaInterpreter.getCurrent;
```

Comment:
any reason for the static import here? It's only used once.

Reply:
Indeed, I remove it